diff --git a/node_modules/@thiagoelg/node-printer/binding.gyp b/node_modules/@thiagoelg/node-printer/binding.gyp
index 3bf38cd..13ad3a7 100644
--- a/node_modules/@thiagoelg/node-printer/binding.gyp
+++ b/node_modules/@thiagoelg/node-printer/binding.gyp
@@ -61,9 +61,10 @@
             "-stdlib=libc++"
           ],
           'xcode_settings': {
-            "OTHER_CPLUSPLUSFLAGS":["-std=c++14", "-stdlib=libc++"],
+            "OTHER_CPLUSPLUSFLAGS":["-std=c++17", "-stdlib=libc++"],
             "OTHER_LDFLAGS": ["-stdlib=libc++"],
-            "MACOSX_DEPLOYMENT_TARGET": "10.7",
+            "MACOSX_DEPLOYMENT_TARGET": "10.13",
+            "GCC_ENABLE_CPP_EXCEPTIONS": "YES",
           },
         }],
       ]
